h/static/scripts/test/root-thread-test.js

```diff
@@ -179,6 +181,53 @@ describe('rootThread', function () {
     ]);
   });
 
+  describe('when the thread filter query is set', function () {
+    it('generates a thread filter function to filter out annotations', function () {
+      fakeBuildThread.reset();
+      fakeBuildThread.annotation = {target: [{ selector: {} }]};
+
+      fakeAnnotationUI.state = Object.assign({}, fakeAnnotationUI.state,
+        {selectedTab: 'annotation'});
+      fakeAnnotationUI.TAB_ANNOTATIONS = 'annotation';
```

Comment:
Rather than re-defining these constants in each test, just import them once when setting up `fakeAnnotationUI`